lib/displayHelpers.js

```diff
@@ -122,9 +127,11 @@ const checkAddress = (input) => {
  * Returns a link to a transaction in an explorer if an explorer is configured
  * for transactions. Returns null otherwise.
  */
-const explorerLinkTx = (hash) => {
-  if (process.env.NEXT_PUBLIC_EXPLORER_LINK_TX) {
-    return process.env.NEXT_PUBLIC_EXPLORER_LINK_TX.replace("%s", hash);
+const explorerLinkTx = (link, hash) => {
+  if (link && link.includes("${txHash}")) {
+    return link.replace("${txHash}", hash);
+  } else if (link && !link.includes("${txHash}")) {
+    return link + hash;
```

Comment:
Do such cases exist? It would be good to only have one pattern. If it is mixed, we should get it standardized in the chain registry.

Reply:
I came across one that didn't have it (agoric), but this wouldn't work in that situation either, so we should remove it.